### PR TITLE
Option to enable or disable the service suffix 

### PR DESCRIPTION
**Pull request by anatosun**
Accordingly to #39

### CODE REVIEW

File: plex-playlist-sync/utils/spotify.py [hunk 1]
```diff
@@ -96,7 +96,8 @@ def spotify_playlist_sync(
         user_id (str): spotify user id
         plex (PlexServer): A configured PlexServer instance
     """
-    playlists = _get_sp_user_playlists(sp, userInputs.spotify_user_id)
+    playlists = _get_sp_user_playlists(
+        sp, userInputs.spotify_user_id, " - Spotify" if userInputs.append_service_suffix else "")
```
Comment (rnagabhyrava) [inline plex-playlist-sync/utils/spotify.py:100]: Please change this to follow 79-character limit like in rest of the code. I use black formatting btw. Sorry, I haven't added a Contibute.md

Ex:
```
    playlists = _get_sp_user_playlists(sp, userInputs.spotify_user_id)
    playlists = _get_sp_user_playlists(
        sp,
        userInputs.spotify_user_id,
        " - Spotify" if userInputs.append_service_suffix else "",
    )
```

File: plex-playlist-sync/utils/deezer.py [hunk 2]
```diff
@@ -102,7 +102,8 @@ def deezer_playlist_sync(
         dz (deezer.Client):  Deezer Client (no credentials needed)
         plex (PlexServer): A configured PlexServer instance
     """
-    playlists = _get_dz_playlists(dz, userInputs)
+    playlists = _get_dz_playlists(
+        dz, userInputs, " - Deezer" if userInputs.append_service_suffix else "")
```
Comment (rnagabhyrava) [inline plex-playlist-sync/utils/deezer.py:106]: Same as the comment for deezer. Use black formatting with 79-character limit.

```
    playlists = _get_dz_playlists(
        dz, userInputs, " - Deezer" if userInputs.append_service_suffix else ""
    )
```

**Comment by rnagabhyrava** [comment]
Thanks for the PR. Keep in mind that if playlists from two services have same name the playlist first created will be overwritten.

**Comment by fwicht** [comment]
Ok I fixed the 79-char limit. Should I add a warning in the README for the eventual conflict when playlists on different services have the same name?

**Review by rnagabhyrava — APPROVED** [review]

**Comment by rnagabhyrava** [comment]
Pushed image to rnagabhyrava/plexplaylistsync:develop. Will create a release and push to latest image after a couple changes I am working on. Next time open a PR onto develop branch. Thanks again for the PR. Cheers!

**Comment by fwicht** [comment]
Noted! Thank's to you for this nice project!

**Comment by rnagabhyrava** [comment]
Don't worry about the warning. I will add it when I publish next release.

**Pull request merged**